modules/tensor_mechanics/include/materials/RadialReturnCreepStressUpdateBase.h

```diff
@@ -22,6 +22,19 @@ class RadialReturnCreepStressUpdateBase : public RadialReturnStressUpdate
 
   RadialReturnCreepStressUpdateBase(const InputParameters & parameters);
 
+  /**
+   * This class computes strain energy rate density for StrainEnergyRateDensity.
+   * The computed material property is needed for the fracture C(t) integral.
```

Comment:
Use the standard doxygen format for methods:
```
/**
 * Compute the strain energy rate density for this inelastic model for the current step.
 * @param strain_energy_rate_density The computed strain energy rate density
 * @param stress The stress tensor at the end of the step
 * @param strain_rate The strain rate at the end of the step
 */
```
No need to include this for the derived class because it will be marked `override`, by the way.

Reply:
Also, seems like it would be better if this method returned a Real. Is there a reason to not do that?

Reply:
Switched to non-`void`